lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -191,6 +204,22 @@ local make_entry = function(opts)
       return raw
     end
 
+    if k == "file_status" then
```

Comment:
Similar comment to above, here `git_status` should suffice because now it's associated with the entry directly.